components/docs-chef-io/content/habitat/studio.md

```diff
@@ -33,73 +22,90 @@ The desire to build Habitat packages in CI leads to the question of why we requi
 
 In order to explain why that is, so that there is a common starting point, my goal is to break down the studio conceptually, and give a brief overview of the differing implementations before talking about the elevated privileges conundrum. -->
 
-## What is the studio
+{{< readfile file="content/habitat/reusable/md/habitat_studio_overview.md" >}}
+
+## Customizing Studio
```

Comment:
Not sure when this happened but we used to have docs on customizing the windows studio that seems to have been removed. See the original PR for the docs on that https://github.com/habitat-sh/habitat/pull/7618/files#diff-b4f52172c4e9339a64b98b03edfabe854ab0af5a78ede124794c014655f2fb79L53